Misc/stable_abi.txt

```diff
@@ -42,6 +42,15 @@
 #   - a combination of the above (functions that were called by macros that
 #     were public in the past)
 
+# For structs, one of the following must be set:
+# - opaque: The struct name is available in the Limited API, but its fields
+#   are not. Users must manipulate it via pointers.
+# - fields: names space-separated fields which are part of the Limited API
```

Comment:
I found that a little hard to read. Maybe something like this?

```suggestion
# - fields: Space-separated list of fields which are part of the Limited API
```

Nit: I also wonder if it would be more consistent to name it `members` iso. `fields`.